nss-cohort-40/banagzon-workforce-mgt-lumberjacks
# De delete computer

# Description
User is able to delete a computer from the detail view if that computer is unassigned.
 ## Type of change
 # Testing Instructions
Test by going to computer list and selecting a computer to view the details on.  If the computer has an assignment, that assignment will be displayed and no delete button should be visible.  If the computer does not have an assignment, you should see a delete button that when clicked makes a dialog box pop up that confirms your choice.  If you then delete, you should be redirected back to the list view.
 # Checklist:
